selenium_tests/data_util.py

```diff
@@ -50,13 +50,13 @@ def create_user_for_login(is_staff=True, username=None):
             image=None,  # make these None so the default image is used
             image_small=None,
             image_medium=None,
-            **({'user_username': username} if username is not None else {}),
             user__social_auth__extra_data={
                 'access_token': 'fake',
                 'refresh_token': 'fake',
                 'updated_at': later.timestamp(),
                 'expires_in': 3600,
-            }
+            },
+            **({'user__username': username} if username is not None else {}),
```

Comment:
possibly dumb question: is there a reason this was moved from line 53 to line 59?

Reply:
there was a warning that python 3.6 do not support **() operator in middle.

Reply:
<img width="454" alt="screen shot 2018-04-10 at 7 28 42 pm" src="https://user-images.githubusercontent.com/10431250/38563075-7757aec0-3cf5-11e8-9724-f5765f946b1b.png">

Reply:
That warning says `<` not `+`. We are using Python 3.6 and are not planning on going backward so I think we should move the line back where it was

Reply:
reverted @noisecapella